Irina/src/main/java/oop/AdminUser.java

```diff
@@ -0,0 +1,27 @@
+package oop;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class AdminUser extends AnyUser {
```

Comment:
What is the purpose of AnyUser class?

Reply:
Removed, logic moved to Abstract user class